tox.ini

```diff
@@ -11,10 +10,28 @@ deps =
 commands = python setup.py test
 
 [testenv:coverage]
+allowlist_externals = coverage
 basepython = python3.8
 deps =
     {[testenv]deps}
-    coveralls
+    ; coveralls
 commands =
     coverage run --source=tapioca setup.py test
-    coveralls
+    ; coveralls
+
+[gh-actions]
+python =
+    3.6: py36
+    3.7: py37
```

Comment:
Drop support from out-of-support Python versions, but note this will need a new release (which is needed anyway because of arrow dependency change).